app/Console/Commands/stubs/task.stub

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace DummyNamespace;
```

Comment:
Yes, it's only template. [Laravel](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Foundation/Console/stubs/request.stub) use this naming so I used too.